WalletWasabi.Gui/Shell/Commands/HelpCommands.cs

```diff
@@ -30,6 +30,22 @@ public HelpCommands(CommandIconService commandIconService, AvaloniaGlobalCompone
 					IoC.Get<IShell>().AddOrSelectDocument(() => new AboutViewModel(Global));
 				}));
 
+			StatusPageCommand = new CommandDefinition(
+			"Status Page",
+			commandIconService.GetCompletionKindImage("StatusPage"),
+			ReactiveCommand.Create(() =>
+			{
+				try
+				{
+					IoHelpers.OpenBrowser("https://status.wasabiwallet.io/");
```

Comment:
I am not sure about this because Wasabi servers should only be accessed through Tor. This could introduce suspicions about Wasabi logging IP addresses.

Reply:
That's a good point. 

In https://github.com/zkSNACKs/WalletWasabi/issues/1559#issuecomment-506979629 uptime robot was suggested, which is a wildly used, generic service. This whould not harm the privacy.

Further, the link is only loaded on demand (not in the background, polled, etc.).

Reply:
Use this link. I finalized the status site:
https://stats.uptimerobot.com/W7q65in4y

Reply:
@molnard @nopara73 the PR is updated and the test passed. It's ready to merge.